packages/cli/index.ts

```diff
@@ -115,14 +115,14 @@ const main = defineCommand({
 
           for (const p of paths) {
             const pJsonPath = path.resolve(p, "package.json");
-            const { name } = await importPackageJson(pJsonPath);
+            const { name } = await readPackageJSON();
```

Comment:
Since we have multiple paths for multiple projects, there should be a way to pass the pJsonPath to this function!